boards/arm/mimxrt1010_evk/init.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2022 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ #include <soc.h>
+
+void SystemInitHook(void)
+{
+#ifdef CONFIG_CODE_FLEXSPI
```

Comment:
Do we want to use this config as eventually I think we are looking to get rid of these configs